src/js/Site.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import ProductList from './components/productList/ProductList.jsx'
+import CurrentFilterInfo from './components/currentFilterInfo/CurrentFilterInfo.jsx'
+import FilterList from './components/filterList/FilterList.jsx'
+import ProductListFooter from './components/productListFooter/ProductListFooter.jsx'
+import ProductListViewSettings from './components/productListViewSettings/ProductListViewSettings.jsx'
+
+export default function Site() {
+    return (
+        <div className="site">
+            <div className="filterColumn">
+                <FilterList className="filterList"/>
```

Comment:
tutaj nie musisz precyzować className - przesyłasz to jako props. Ważne aby w metodzie render komponentu FilterList to się znalazło. No chyba, że tam używasz <div className={this.props.className} ale nie chciałbym abyśmy tak to robili bo:
chcemy aby komponent miał zawsze przypisany stały className na potrzeby stylowania. Wyjątkiem była by jednak taka reguła: <div className={"FilterList " + this.props.className}...